pkg/gotohelm/transpiler.go

```diff
@@ -607,6 +607,10 @@ func (t *Transpiler) transpileCallExpr(n *ast.CallExpr) Node {
 	name := callee.Pkg().Name() + "." + callee.Name()
 
 	if tplFuncName, ok := funcMapping[name]; ok {
+		if tplFuncName == "merge" {
```

Comment:
Could you push this change into the switch statement below? That's where all the mappings are for non-one-to-one functions are. See `helmette.Dig` for an example :D

Reply:
Ok, I will do that.